src/components/Button/Button.tsx

```diff
@@ -27,6 +27,7 @@ export interface ButtonProps extends ButtonNextProps, TPAComponentProps {
   size?: SIZE;
   fullWidth?: boolean;
   innerRef?: React.RefObject<HTMLButtonElement>;
+  upgrade?: boolean;
 }
 
 class ButtonComponent extends React.Component<ButtonProps> {
```

Comment:
We also need to add a `console.warn` with a warning if consumers aren't using the `upgrade` prop.
You can see the implementation of the `Card` component.